File: 27-9/Apiinterface.java
package com.example.retrofitex;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.GET;
import retrofit2.http.POST;

public interface Apiinterface
{
   @FormUrlEncoded
   @POST("productinsert.php")
   Call<Void>productadd(
           @Field("product_name")  String prodct_name,
           @Field("product_price") String product_price,
           @Field("product_description") String product_description
   );

   @GET("productview.php")
   Call<List<Model>> productview();


   @FormUrlEncoded
   @POST("productupdate.php")
   Call<Void>productupdate(
           @Field("product_id")  int product_id,
           @Field("product_name")  String prodct_name,
           @Field("product_price") String product_price,
           @Field("product_description") String product_description
   );
   @FormUrlEncoded
   @POST("productdelete.php")
   Call<Void>productdelete
           (
               @Field("product_id")  int product_id

            );
}

File: 10-11/Project/Interface/Apiinterface.kt
package com.example.e_greetingsheet.Interface

import com.example.e_greetingsheet.Model.RegisterModel
import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface Apiinterface
{

    @FormUrlEncoded
    @POST("registration.php")
    fun registerdetail(
        @Field("firstname") firstname: String?,
        @Field("lastname") lastname: String?,
        @Field("phone") mobile: String?,
        @Field("email") email: String?,
        @Field("password") password: String?,

        ): Call<Void>

    @FormUrlEncoded
    @POST("login.php")
    fun logindata(  @Field("phone") mobile: String?,  @Field("pass") password: String?):Call<Void>


}